src/tests/multihost/alltests/test_sssctl_analyzer.py

```diff
@@ -295,3 +295,40 @@ def test_analyze_parse_child_logs(self, multihost, backupsssdconf):
             assert re.findall(r"RID#[0-9]*] Received error code", stdout)
         else:
             pytest.fail(f"{user} sucessful to login")
+
+    @staticmethod
+    def test_non_root_privileged(multihost, backupsssdconf, localusers):
```

Comment:
Does not update the sssd.conf, so we can remove it.

Reply:
Done